src/coreclr/tools/Common/TypeSystem/Interop/IL/Marshaller.cs

```diff
@@ -1603,24 +1597,76 @@ protected override void TransformManagedToNative(ILCodeStream codeStream)
             // ANSI marshalling. Allocate a byte array, copy characters
             //
 
-#if READYTORUN
```

Comment:
do we have a usecase for non `READYTORUN`, or should we cleanup the codebase?

Reply:
These READYTORUN ifdefs is allows easier sharing of the code with the NativeAOT branch.

Reply:
ok, thanks. Have keep `!READYTORUN` as-is then.